prerendering.bs

```diff
@@ -478,172 +459,146 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
+  1. If |traversable| is a [=prerendering traversable=] and |traversable|'s [=prerendering traversable/remove from referrer=] is not null, then call |traversable|'s [=prerendering traversable/remove from referrer=].
 </div>
 
-<h3 id="creating-bcs-patch">Modifications to creating browsing contexts</h3>
+<h3 id="creating-navigables-patch">Modifications to creating navigables</h3>
 
-<div algorithm="create a new nested browsing context patch">
-  To ensure that any [=nested browsing contexts=] inherit their parent's [=browsing context/loading mode=], modify <a spec=HTML>create a new nested browsing context</a> by appending the following step:
+<div algorithm="create a new nested navigable patch">
+  To ensure that any [=child navigables=] inherit their [=navigable/parent=]'s [=navigable/loading mode=], modify [=create a new nested navigable=] by appending the following step:
 
-  1. Set <var ignore>browsingContext</var>'s [=browsing context/loading mode=] to <var ignore>element</var>'s [=Node/node document=]'s [=Document/browsing context=]'s [=browsing context/loading mode=].
+  1. Set <var ignore>navigable</var>'s [=navigable/loading mode=] to <var ignore>element</var>'s [=node navigable=]'s [=navigable/loading mode=].
 </div>
 
 <h2 id="navigation">Navigation and session history</h2>
 
 <h3 id="navigate-activation">Allowing activation in place of navigation</h3>
 
-<div algorithm="can activate a prerender">
-  We <dfn>can activate a prerender</dfn> given a [=browsing context=] |browsingContext|, a [=history handling behavior=] |historyHandling|, a string |navigationType|, and a [=request=] |request|, if the following steps return true:
+<div>
+  To <dfn>get the matching prerendering navigable</dfn> given a [=navigable=] |navigable|, a [=URL=] |url|, a string |cspNavigationType|, and a |documentResource|, if the following steps return true:
 
-  1. Return true if all of the following are true:
+  1. If any of the following conditions hold:
 
-      * |browsingContext| is a [=top-level browsing context=]
-      * |browsingContext| is not a [=prerendering browsing context=]
-      * |browsingContext| is not for a [Fenced Frame](https://github.com/shivanigithub/fenced-frame)
-         <p class="issue">The concept of a fenced frame browsing context is not yet defined but we need to link to it once it exists.</p>
-      * |historyHandling| is "`default`" or "`replace`"
-      * |navigationType| is "`other`"
-      * |request|'s [=request/method=] is \``GET`\`
-      * |browsingContext|'s [=active document=]'s [=Document/prerendering browsing contexts map=][|request|'s [=request/URL=]] [=map/exists=] and is not [=prerendering browsing context/empty=]
-      * The result of calling [=should navigation request of type be blocked by content security policy?=] given |request| and <var ignore>navigationType</var> is "`Allowed`".
```

Comment:
It's not obvious to me why this was removed; can you clarify?

Reply:
Now that `navigate-to` is dead, this only matters for CSP `form-action` and `javascript:` URLs. We're already excluding those more generally, so I removed it.

This is not perfectly future proof, so maybe I should add it back. But:

- Doing so would be a bit of a pain, as we'd need to construct a synthetic response (since that's what CSP expects)
- It's arguably reasonable to block such navigations at prerendering time, and not necessarily check a second time when activating. Maybe?

I'll add it back if you don't find these excuses convincing :)